cms/lib/xblock/tagging/tagging.py

```diff
@@ -43,7 +43,8 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
         Display the tag selector with specific categories and allowed values,
         depending on the context.
         """
-        if isinstance(block, CapaModule):
```

Comment:
@andy-armstrong, @cahrens: Can one of you comment on the difference between `is_author_mode` and `author_view` in Studio?